build.sbt

```diff
@@ -238,9 +238,9 @@ lazy val `sbt-shared` = project
   .settings(
     plugin,
     utest,
-//  addSbtPlugin("com.dwijnand" % "sbt-compat" % "1.2.0+2-f30b82f4")
+//  addSbtPlugin("com.dwijnand" % "sbt-compat" % "1.2.1")
```

Comment:
could you remove this line, or add a comment why this comment is needed?

Reply:
Sure.